package.json

```diff
@@ -37,7 +37,7 @@
     "test": "TZ=utc NODE_OPTIONS=--experimental-vm-modules jest",
     "typecheck": "NODE_OPTIONS=--max_old_space_size=8192 tsc",
     "lint": "NODE_OPTIONS=--max_old_space_size=8192 eslint . --max-warnings=0 --cache --cache-location=node_modules/.cache/eslint",
-    "lint-changed": "eslint --fix $(git diff --diff-filter=AM --name-only main -- \"*.js\" \"*.ts\" \"*.tsx\")",
+    "lint-changed": "NODE_OPTIONS=--max_old_space_size=8192 eslint --max-warnings=0 --config ./.eslintrc.changed.js $(git diff --diff-filter=AM --name-only origin/main HEAD -- \"*.ts\" \"*.tsx\")",
```

Comment:
@blazejkustra  I see you remove js from this command. Should we also clean up ".js" in main eslintrc file
 https://github.com/Expensify/App/blob/e52d43c2cd8ae7d8449161651867e8b163ac1321/.eslintrc.js#L273

Reply:
@blazejkustra @roryabraham Should we only run lint and prettier on the changed file by using the same way: `$(git diff --diff-filter=AM --name-only origin/main HEAD --`

Reply:
> @blazejkustra I see you remove js from this command. Should we also clean up ".js" in main eslintrc file
> 
> https://github.com/Expensify/App/blob/e52d43c2cd8ae7d8449161651867e8b163ac1321/.eslintrc.js#L273

I think we should lint all the files, no matter if it's JS or TS

Reply:
IMO When linting changed files we can omit JS, however if you think otherwise I can add it back 😄